lib/rackup/handler/webrick.rb

```diff
@@ -93,18 +93,23 @@ def service(req, res)
 
         status, headers, body = @app.call(env)
         begin
-          res.status = status.to_i
-          io_lambda = nil
+          res.status = status
+
+          if value = headers[RACK_HIJACK]
```

Comment:
Should we delete this header after it's extracted?

```suggestion
          if value = headers.delete(RACK_HIJACK)
```

Reply:
I don't think we should modify the hash.

Reply:
I say this, and look at the code and see it currently deletes for `set-cookie` :)  Still, if we are skipping all keys starting with rack, doesn't seem necessary to delete here.  I'm not against it if you want to commit it, though.

Reply:
I'm fine to ignore it here, just wanted to present the options.